cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -2588,8 +2631,13 @@ declare namespace Deno {
    * @category I/O
    */
   export const stdout: Writer & WriterSync & Closer & {
-    /** The resource ID assigned to `stdout`. This can be used with the discreet
-     * I/O functions in the `Deno` namespace. */
+    /**
+     * The resource ID assigned to `stdout`. This can be used with the discreet
+     * I/O functions in the `Deno` namespace.
+     *
+     * @deprecated Use instance methods directly instead.
+     * {@linkcode Deno.stdout.rid} will be removed in v2.0.0.
+     */
```

Comment:
Should we prevent the explicit construction of an `FsFile`?

Reply:
yes

Reply:
Will do.

Reply:
Done.